TelepresenceBot/core/src/main/java/com/novoda/tpbot/human/HumanPresenter.java

```diff
@@ -23,8 +23,13 @@ void startPresenting(String serverAddress) {
     }
 
     void stopPresenting() {
-        observable.detachObservers();
-        humanTpService.disconnect();
+        if (observable != null) {
```

Comment:
Shouldn't this be DeveloperError ? When you `stopPresenting` before calling `startPresenting` ?